emperor/support_files/js/view.js

```diff
@@ -37,16 +37,19 @@ function DecompositionView(decomp, asPointCloud) {
   /**
    * Top visible dimensions
    * @type {integer[]}
-   * @default [0, 1, 2]
    */
-  this.visibleDimensions = [0, 1, 2]; // We default to the first three PCs
+  // this array should have at most three elements
+  this.visibleDimensions = _.range(this.decomp.dimensions).slice(0, 3);
```

Comment:
Based on the comment, I feel like we need a if/else or perhaps I'm not clear?

Reply:
The way `slice` works allows us to not need an if/else. For example if the number of dimensions is 10 `_.range` is going to produce `[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]` and `slice` will reduce this to `[0, 1, 2]`. In the case where the number of dimensions is 2 (what originated this pull request) the output of `_.range` will be `[0, 1]` and `slice` will keep this is as `[0, 1]` giving the correct number of visible dimensions.